com.Acestream.engine.json

```diff
@@ -0,0 +1,149 @@
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "https://github.com/pygobject/pycairo/releases/download/v1.18.1/pycairo-1.18.1.tar.gz",
+                            "sha256": "70172e58b6bad7572a3518c26729b074acdde15e6fee6cbab6d3528ad552b786"
+                        }
+                    ]
+                },
+                {
+                    "name": "pygobject",
+                    "config-opts": ["--disable-glibtest"],
+                    "cleanup": ["/bin"],
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "https://download.gnome.org/sources/pygobject/2.28/pygobject-2.28.7.tar.xz",
+                            "sha256": "bb9d25a3442ca7511385a7c01b057492095c263784ef31231ffe589d83a96a5a"
+                        }
+                    ]
+                }
+            ]
+        },
+            "libs/libappindicator/libappindicator-gtk2-12.10.json",   
+         {
+            "name": "acestream",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm755 apply_extra /app/bin/apply_extra",
+                "install -Dm755 acestream.sh /app/bin/acestream",
+                "for size in 16 24 32 48 64 128 256; do\n                    rsvg-convert -w $size -h $size -f png -o $size.png logo-small.svg\n                    install -Dm644 $size.png /app/share/icons/hicolor/${size}x${size}/apps/com.Acestream.engine.png\n                done",
+                "install -Dm644 logo-small.svg export/share/icons/hicolor/scalable/apps/com.Acestream.engine.svg",
+                "install -Dm644 com.Acestream.engine.appdata.xml /app/share/appdata/com.Acestream.engine.appdata.xml",
+                "install -Dm644 com.Acestream.engine.desktop /app/share/applications/com.Acestream.engine.desktop"
+            ],
+            "sources": [
+                {
+                    "type": "script",
+                    "dest-filename": "apply_extra",
+                    "commands": [
+                        "tar -zxvf acestream.tar.gz",
+                        "rm -f acestream.tar.gz"
+                    ]
+                },
+               {
+                    "type": "script",
+                    "dest-filename": "acestream.sh",
+                    "commands": [
+                        "exec /app/extra/start-engine --client-gtk --log-file $XDG_DATA_HOME/acestream.log \"$@\""
+                    ]
+                },                                                                            
+                {
+                    "type": "extra-data",
+                    "filename": "acestream.tar.gz",
+                    "only-arches": [
+                        "x86_64"
+                    ],
+                    "url": "https://drive.google.com/uc?id=1hIWE_hrxnmBiQWNRJYTAKJCgtI0lwdcr&export=download",
```

Comment:
Doesn't sound like a source I would trust.

Reply:
"Error: While downloading http://download.acestream.media/linux/acestream_3.1.49_ubuntu_18.04_x86_64.tar.gz: Server returned status 404: Not Found" - So I had to transfer to google drive.

Reply:
I just don't want users to wait 40+ minutes when http://download.acestream.media/linux/acestream_3.1.49_ubuntu_18.04_x86_64.tar.gz downloads the file.

Reply:
I can't trust a random google drive link, you should ask upstream to provide another download source?

Reply:
Returned the old link.